node-manager/src/keymanager.rs

```diff
@@ -69,7 +69,7 @@ mod tests {
             second_pubkey.to_string()
         );
 
-        let km = create_keys_manager(mnemonic.clone(), 2);
```

Comment:
I wish something would tell me that I didn't need to `clone()`. Sometimes those linger around for me.

Reply:
cargo clippy sometimes catches these